docs/design/gh-ost-integration.md

```diff
@@ -267,35 +269,35 @@ type Instance struct {
 
 #### Or we speculate
 
-Another approach is to infer cloud providers by inspecting usernames because cloud providers typically will have some default accounts to manage databases. References below.
+Another approach is to infer cloud providers by inspecting usernames because cloud providers typically will have some default accounts to manage databases. References are below.
 
 - [GCP](https://cloud.google.com/sql/docs/mysql/users)
 - [aliyunrds](https://help.aliyun.com/document_detail/161152.html)
 - [azure](https://docs.microsoft.com/en-us/azure/mysql/howto-create-users)
 - [aws](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.MySQL.CommonDBATasks.html) (Stored procedures)
 
-We will take the first approach, this section is left here for information.
+We will take the first approach, and this section is left here for information.
 
 ### gh-ost execution stages
 
-We will divide gh-ost execution into different tasks. Each task needs to be approved by user to proceed. These tasks are described in detail in the following sections.
+We will divide gh-ost execution into different tasks. Each task needs to be approved by the user to proceed. These tasks are described in detail in the following sections.
 
-1. Pre-migration check. We run gh-ost with noop flag to check user's input.
+1. Pre-migration check. We run gh-ost with a no-op flag to check the user's input.
 2. In-migration,
-    1. After approval, gh-ost performs the migration, syncs the ghost table with the original table.
-    2. After the ghost table is synced with the original table, there is a cut-over phase. We will only allow gh-ost to do the cut-over if user approved.
-3. After migration, the final task is to delete the original table if user approved.
+    1. After approval, gh-ost performs the migration and syncs the ghost table with the original table.
+    2. There is a cut-over phase when the ghost table is in sync with the original table. We will only allow gh-ost to do the cut-over if user-approved.
+3. After migration, the final task is to delete the original table if the user approves.
 
 ### Pre-migration validation
 
-gh-ost has many requirements and limitations, we want to check whether gh-ost is good to go before the actual migration. To validate it, we run gh-ost with noop flag. We will return the error, if any, to user and let user handle it. The error includes but not limited to the followings
+gh-ost has many requirements and limitations. We want to check whether gh-ost is good to go before the actual migration. To validate it, we run gh-ost with a no-op flag. If any, we will return the error to the user and let the user handle it. The error includes but is not limited to the following.
 
 - SQL statement not legal
 - `_tablename_ghc` present
 - `_tablename_gho` present
 - binlog format is not row based.
 
-This will be a new task check run type.
+The following will be a new task check run type.
```

Comment:
```suggestion
The above will be a new task check run type.
```

Reply:
If you're referencing `TaskCheckDatabaseSchemaUpdateGhost`, I suppose it's *below* or *following*?

Reply:
Yes, I am referencing `TaskCheckDatabaseSchemaUpdateGhost`. "Above" means the above discussion entails such type in the following. I guess it's ambiguous here.

Reply:
Updated the suggestion.